ts/twitter/src/lib/theme.ts

```diff
@@ -13,6 +13,7 @@ const theme = {
     blue: {
       primary: "#1DA1F2",
       primaryHover: "#1A91DA",
+      secondary: "#1D9BF0",
```

Comment:
```suggestion
      primary: "#1D9BF0",
      primaryHover: "#1A91DA",
```

https://github.com/okuda-seminar/Twitter-Clone/pull/609#discussion_r2000870250